tests/testthat/test-type-data-frame.R

```diff
@@ -281,9 +281,10 @@ test_that("attributes with special names are merged", {
     "rowname"
   )
 
-  expect_error(
+  # Can omit n if row.names attribute is given
```

Comment:
`n` is not omitted in this test. Should probably be its own `test_that()` block.